schnorr_fun/Cargo.toml

```diff
@@ -44,7 +45,7 @@ harness = false
 
 [features]
 default = ["std"]
-all = ["std","serde", "libsecp_compat", "proptest"]
+all = ["std", "serde", "libsecp_compat", "proptest"]
```

Comment:
removed whitesspace

Reply:
I'm adding whitespace here, i think it was an rustfmt